test/compress/export.js

```diff
@@ -154,3 +154,46 @@ issue_2134_2: {
         Foo.prototype = {};
     }
 }
+
+redirection: {
+    mangle = {
+        toplevel: true,
+    }
+    input: {
+        let foo = 1;
+        export { foo as var };
+        export { foo as var } from "module.js";
```

Comment:
> Or does default has a special meaning as an export alias?

In your logic as a property name it does not have special meaning, but for ES6 programs it does. Just covering all the bases.